easybuild/easyconfigs/t/TAMkin/TAMkin-1.2.4-intel-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = "PythonPackage"
+
+name = 'TAMkin'
+version = '1.2.4'
+versionsuffix = "-Python-%(pyver)s"
+
+homepage = 'http://molmod.github.io/tamkin/'
+description = """TAMkin is a post-processing toolkit for normal mode analysis,
+ thermochemistry and reaction kinetics. It uses a Hessian computation from a
+ standard computational chemistry program as its input."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
```

Comment:
Hmm, this is the only one using `intel/2016a` rather than `intel/2017b`, does it belong to the same batch of software?

Reply:
No, this is an older one, in fact unrelated to the other ones. I haven't converted TAMkin to work with Python 3 yet.